src/app/lightning/get-lightning-fee.ts

```diff
@@ -0,0 +1,80 @@
+import { checkedToSats, toSats } from "@domain/bitcoin"
+import {
+  decodeInvoice,
+  LnFeeCalculator,
+  RouteNotFoundError,
+} from "@domain/bitcoin/lightning"
+import { LnPaymentRequestZeroAmountRequiredError } from "@domain/errors"
+import { CachedRouteLookupKeyFactory } from "@domain/routes/key-factory"
+import { LndService } from "@services/lnd"
+import { RoutesCache } from "@services/redis/routes"
+
+export const lnInvoiceFeeProbe = async ({
+  paymentRequest,
```

Comment:
this use case needs to add amount validation so we will require here wallePublicId and move this use case to wallets folder.

@nicolasburtey mentions this validation as a possible attack vector and graphql already has the walletId https://github.com/GaloyMoney/galoy/pull/752

Reply:
Added the user balance check and moved to `wallets` dir